.github/renovate.json5

```diff
@@ -1,7 +1,20 @@
 {
-  enabledManagers: ["github-actions", "gomod"],
-  postUpdateOptions: [
-    "gomodTidy", // go mod tidy
-    "gomodUpdateImportPaths",
+  enabledManagers: ["github-actions", "gomod", "regex"],
+  postUpdateOptions: ["gomodTidy", "gomodUpdateImportPaths"],
+  regexManagers: [
+    {
+      fileMatch: ["Dockerfile$"],
+      matchStrings: [
+        "datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\n.*_VERSION=(?<currentValue>.*)",
```

Comment:
I see this relates to line 9 of the Dockerfile. I assume the point is to keep the version number on line 10 up to date. How will that be achieved? I'm having trouble figuring out what part of this config will say "update the `ARG TINYGO_VERSION=<version>` to whatever the latest version of `tinygo-org/tinygo` is"

Reply:
It's kinda magical, but it uses [this](https://docs.renovatebot.com/modules/manager/regex/). It depends on the capture group `currentValue` to determine what the version is and replaces it, if the remote value is larger semver.

Reply:
right, I see. So it checks for two lines. One is this one:
```
# renovate: datasource=github-releases depName=tinygo-org/tinygo
```
and the other line is one immediately following that which has a `_VERSION=semver` in it somewhere, and it's kinda strict that these two go together, and that the line above relates to the version below